assets/rpip-53/rewards-file-spec.md

```diff
@@ -0,0 +1,103 @@
+# Specification for Rewards Tree File
```

Comment:
this was fixed (File -> Files) upstream - did you want to change it back, or just need to rebase?

Reply:
Yep, rebased with the latest asset from RPIP-52 - thanks!